src/components/apps/StoreHeader.jsx

```diff
@@ -4,9 +4,25 @@ import { FaEdit } from 'react-icons/fa';
 import { FiUser } from 'react-icons/fi';
 import { Link } from 'react-router-dom';
 import { lozodo_logo } from '../../assets/images';
-import { PRODUCT_DASHBOARD } from '../../constants/routes';
+import { PATH_CART, PRODUCT_DASHBOARD } from '../../constants/routes';
+import { useAuthenticationStore, useCartStore } from '../../store';
+import { useEffect } from 'react';
+import { JWTStorage } from '../../utils';
+import { userService } from '../../services';
+import { useCartHooks } from '../../hooks';
 
 const StoreHeader = () => {
+	const { itemsInCart } = useCartStore((state) => state);
+	const { setAuthenticatedUser } = useAuthenticationStore((state) => state);
+	useCartHooks();
```

Comment:
what is this for?

Reply:
itemsInCart to display the number of items in Cart Badge.

setAuthenticatedUser used to set the authenticated user on first load if there is already a stored token on localStorage.

useCartHooks() to load to carts from api/local storage depending on if user is logged in or not.

Reply:
@AccentureAngelo heads up. Pa-review ako ng implementation kapag nasa sayo na yung PR.